lib/src/bloc/dice_bloc.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:async';
+import 'dart:math';
+import 'package:flutter/material.dart';
+
+import '../screen/dice/dice_event.dart';
+import 'package:first_flutter_app/src/bloc/bloc.dart';
```

Comment:
Use relative imports for your own classes/files

Reply:
Done!

Reply:
this one still needs to be fixed